server/services/GithubService.js

```diff
@@ -29,28 +29,30 @@ const permissionScopes = [
 ];
 const permissionScopesURI = encodeURI(permissionScopes.join(' '));
 const graphQLEndpoint = `${GITHUB_GRAPHQL_SERVER}/graphql`;
-const nodeCache = new NodeCache();
 
-const getAllIssuesFromGitHub = async () => {
+const getAllIssues = async () => {
     let currentResults = [];
     let page = 1;
 
     // eslint-disable-next-line no-constant-condition
     while (true) {
-        const issuesEndpoint =
-            `${GITHUB_ISSUES_API_URL}/issues` +
-            `?labels=app&state=all&per_page=100`;
+        const issuesEndpoint = `${GITHUB_ISSUES_API_URL}/issues?state=all&per_page=100`;
```

Comment:
Was this the only place where the `app` label was being used? If that's the case, you could stop it being added during `createIssueLink`, to have less 'noise' in the labels. I suppose it could be pretty useful to quickly note what was created by the app, but I also have no strong feelings for either.